test/helpers.js

```diff
@@ -0,0 +1,27 @@
+export default {
+	waitForCssRules: function (styleNode, callback) {
+		if (typeof callback !== 'function') {
+			return;
```

Comment:
Yeah, but that would be a mistake, right? If so I think it's better to throw an error, otherwise you get no feedback that you screwed up.